css/selectmenu.css

```diff
@@ -163,19 +164,24 @@
 .toplevel_page_mlang .ui-selectmenu-button.ui-selectmenu-button-open{ /* To be compatible jQuery UI 1.12.1 since WordPress 5.6 */
 	box-shadow: 0 0 0 transparent;
 	border-radius: 4px;
-    border: 1px solid #7e8993;
+	border: 1px solid #7e8993;
 }
 
 .toplevel_page_mlang .ui-selectmenu-button:focus{
+	color: #016087; /* Same color as WordPress focused select HTML tag */
 	border-color: #007cba;
-    box-shadow: 0 0 0 1px #007cba;
-    outline: 2px solid transparent;
+	box-shadow: 0 0 0 1px #007cba;
+	outline: 2px solid transparent;
+}
+
+.toplevel_page_mlang .ui-menu-item {
+	color: #016087; /* Same color as option in a WordPress focused select HTML tag */
 }
 
 .ui-widget-content .ui-state-hover,
 .ui-widget-content .ui-state-focus,
 .ui-widget-content .ui-state-active { /* To be compatible jQuery UI 1.12.1 since WordPress 5.6 */
-	background: #f5f5f5;
+	background: #d5d5d5;
```

Comment:
Yes! only for this rule. `.ui-state-active` is a new CSS class used by jQuery UI 1.12.1 instead of `.ui-state-hover` and `.ui-state-focus` used by jQuery UI 1.11.4. 
This comment already exists in the previous PR #609 see [here](https://github.com/polylang/polylang/pull/609/files#r525878095)